import inital version of python styleguide (#1)
exasol/python-styleguide: README.rst

Exasol python-styleguide
========================
Welcome to the Exasol_ python-styleguide.

> The styleguide(s) is not intend to be a universal text on how to do each single detail in python,

Reviewer: ```suggestion
The styleguide(s) is not intend to be an universal text on how to do each single detail in python,
```

> it rather tries to reley on existing documents and guides references them (e.g. pep8_) and their rules or it

Reviewer: ```suggestion
it rather tries to rely on existing documents and guides references them (e.g. pep8_) and their rules or it
```

> will add useful additions or exceptions where Exasol_ is deverging from the *"norm"* and why.

Reviewer 2: ```suggestion
will add useful additions or exceptions where Exasol_ is diverging from the *"norm"* and why.
```

> will add useful additions or exceptions where Exasol_ is diverging from the *"norm"* and why.

Reviewer: ```suggestion
will add useful additions or exceptions where Exasol_ is diverging from the *"norm"* and will explain the reasons behind.
```
